[IMP] stock_release_channel_shipment_advice: Add buttons on release channel kanban to print shipments and delivery slip

tuantrantg opened the pull request.

sbejaoui committed:
[ADD] stock_release_channel_shipment_advice_toursolver

Use TourSolver to plan shipment advices for ready and released pickings.

sbejaoui committed:
[FIX] stock_release_channel_shipment_advice_toursolver: fix method name

sbejaoui committed:
[IMP] stock_release_channel_shipment_advice_toursolver: set release channel name as toursolver task ref

tuantrantg committed:
[IMP] stock_release_channel_shipment_advice: Add buttons on release channel kanban to print shipments and delivery slip

nilshamerlinck commented on stock_release_channel_shipment_advice_print_cash_on_delivery/__manifest__.py:
```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
```
```suggestion
# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
# Copyright 2024 Camptocamp SA
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

nilshamerlinck commented on stock_release_channel_shipment_advice_print_cash_on_delivery/models/shipment_advice.py:
```diff
@@ -0,0 +1,20 @@
+class ShipmentAdvice(models.Model):
+    _inherit = "shipment.advice"
+
+    def print_cash_invoices(self):
```
```suggestion
    def print_cash_on_delivery_invoices(self):
```

nilshamerlinck replied:
cash != cod

nilshamerlinck commented on stock_release_channel_shipment_advice_print_cash_on_delivery/models/stock_picking.py:
```diff
@@ -0,0 +1,32 @@
+    _inherit = "stock.picking"
+
+    cash_on_delivery_invoice_ids = fields.Many2many(
+        "account.move", string="Invoice", copy=False, readonly=True
```
```suggestion
        "account.move", string="COD Invoices", copy=False, readonly=True
```

nilshamerlinck commented on stock_release_channel_shipment_advice_print_cash_on_delivery/models/stock_release_channel.py:
```diff
@@ -0,0 +1,14 @@
+class StockReleaseChannel(models.Model):
+    _inherit = "stock.release.channel"
+
+    def action_print_cash_invoices(self):
```
```suggestion
    def action_print_cash_on_delivery_invoices(self):
```

tuantrantg commented:
hi @nilshamerlinck the PR's updated, please review it again, thank you

sebalix committed:
stock_available_to_promise_release: add api.depends hooks for 'release_ready' field

sebalix committed:
[ADD] stock_available_to_promise_release_block

sebalix committed:
[ADD] sale_stock_available_to_promise_release_block

tuantrantg committed:
[ADD] stock_release_channel_shipment_advice_cash_on_delivery: Add buttons on release channel kanban to print cash on delivery invoices

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (139 of 139 strings)

Translation: wms-16.0/wms-16.0-stock_release_channel
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_release_channel/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: wms-16.0/wms-16.0-stock_release_channel_shipment_advice_process_end_time
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_release_channel_shipment_advice_process_end_time/it/

jbaudoux committed:
[ADD] stock_available_to_promise_release_exclude_location

OCA-git-bot committed:
Merge PR #865 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_available_to_promise_release_block.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #885 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update stock_available_to_promise_release_exclude_location.pot

OCA-git-bot committed:
[BOT] post-merge updates

santostelmo committed:
[IMP] stock_available_to_promise_release_block tracking release_blocked

OCA-git-bot committed:
Merge PR #867 into 16.0

Signed-off-by jbaudoux

Commit:
[UPD] Update sale_stock_available_to_promise_release_block.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #887 into 16.0

Signed-off-by jbaudoux

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #577 into 16.0

Signed-off-by rousseldenis

Commit:
[UPD] Update stock_release_channel_shipment_advice_toursolver.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: wms-16.0/wms-16.0-stock_available_to_promise_release_exclude_location
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_available_to_promise_release_exclude_location/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (8 of 8 strings)

Translation: wms-16.0/wms-16.0-stock_release_channel_shipment_advice_toursolver
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_release_channel_shipment_advice_toursolver/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: wms-16.0/wms-16.0-sale_stock_available_to_promise_release_block
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-sale_stock_available_to_promise_release_block/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (15 of 15 strings)

Translation: wms-16.0/wms-16.0-stock_available_to_promise_release_block
Translate-URL: https://translation.odoo-community.org/projects/wms-16-0/wms-16-0-stock_available_to_promise_release_block/it/
